CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ configure_file("${CMAKE_SOURCE_DIR}/cmake/templates/license.h.in" include/licens
 
 include(EthOptions)
 configure_project(TESTS)
-set(LATEST_Z3_VERSION "4.11.2")
+set(LATEST_Z3_VERSION "4.12.1")
```

Comment:
We should really only do this in the second PR we create, s.t. builds don't break here, shouldn't we :-)?

Reply:
Yea we shouldn't merge that here, but without this the build won't go through inside docker